[JIT][AMP] Require full precision for caffe2::PiecewiseLinearTransform

davidberard98 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #72079

Internal model requires caffe2::PiecewiseLinearTransform for AMP

Differential Revision: [D33851676](https://our.internmc.facebook.com/intern/diff/D33851676/)

davidberard98 committed:
[JIT][AMP] Require full precision for caffe2::PiecewiseLinearTransform

Internal model requires caffe2::PiecewiseLinearTransform for AMP

Differential Revision: [D33851676](https://our.internmc.facebook.com/intern/diff/D33851676/)

[ghstack-poisoned]

davidberard98 commented:
Not sure if this is the right solution here - doesn't seem like any other caffe2 ops are used with autocasting, not sure if there's any issues with adding it here?

@eellison thoughts on this?

eellison reviewed (approved):
fine landing this temporarily but we should look into modifying model code

davidberard98 committed:
Update on "[JIT][AMP] Require full precision for caffe2::PiecewiseLinearTransform"


Internal model requires caffe2::PiecewiseLinearTransform for AMP

Differential Revision: [D33851676](https://our.internmc.facebook.com/intern/diff/D33851676/)

[ghstack-poisoned]

davidberard98 committed:
Update on "[JIT][AMP] Require full precision for caffe2::PiecewiseLinearTransform"


Internal model requires caffe2::PiecewiseLinearTransform for AMP

Differential Revision: [D33851676](https://our.internmc.facebook.com/intern/diff/D33851676/)

[ghstack-poisoned]

davidberard98 committed:
Update on "[JIT][AMP] Require full precision for caffe2::PiecewiseLinearTransform"


Internal model requires caffe2::PiecewiseLinearTransform for AMP

Differential Revision: [D33851676](https://our.internmc.facebook.com/intern/diff/D33851676/)

[ghstack-poisoned]
